src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ CHANGELOG
  * added `assertFormValue()` and `assertNoFormValue()` in `WebTestCase`
  * Added "--as-tree=3" option to `translation:update` command to dump messages as a tree-like structure. The given value defines the level where to switch to inline YAML
  * Deprecated the `lock.RESOURCE_NAME` and `lock.RESOURCE_NAME.store` services and the `lock`, `LockInterface`, `lock.store` and `PersistingStoreInterface` aliases, use `lock.RESOURCE_NAME.factory`, `lock.factory` or `LockFactory` instead.
+ * Deprecate the `session` service and the `SessionInterface` alias, use the `Request::getSession()` or the new `RequestStack::getSession()` methods instead
```

Comment:
for 5.3, not 5.2

Reply:
fixed (here and in src/Symfony/Component/HttpFoundation/CHANGELOG.md)